common/src/app/common/types/tokens_lib.cljc

```diff
@@ -1301,22 +1292,32 @@ Will return a value that matches this schema:
 
   (get-tokens-in-active-sets [this]
     (let [theme-set-names  (get-active-themes-set-names this)
-          all-set-names    (get-ordered-set-names this)
+          all-set-names    (get-set-names this)
           active-set-names (filter theme-set-names all-set-names)
           tokens           (reduce (fn [tokens set-name]
-                                     (let [set (get-set this set-name)]
-                                       (merge tokens (get-tokens-map set))))
+                                     (let [set (get-set-by-name this set-name)]
+                                       (merge tokens (get-tokens-map- set))))
                                    (d/ordered-map)
                                    active-set-names)]
       tokens))
 
   (get-all-tokens [this]
     (reduce
      (fn [tokens' set]
-       (into tokens' (map (fn [x] [(:name x) x]) (get-tokens set))))
+       (into tokens' (map (fn [x] [(:name x) x]) (get-tokens-seq- set))))
      {}
      (get-sets this)))
 
+  (get-tokens-seq [this set-id]
```

Comment:
This method is redundant because we can already call `(vals (get-tokens-map ...))` 
I suggest just renaming it to `get-tokens` always returning the map of tokens (the most efficient operation), if user need seq of it, it can call seq on it.

Reply:
Ok. It seems reasonable.